notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1593_force_actuator_error_elevation.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    max_actuators = np.argmax(df_primary_FA_error_resampled_max, axis=1)
</pre><p>It seems that this is only used in a plot a few cells below. Maybe bring line 6 and 7 closer to their associated plot so it is easier to realize where the data is coming from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>